tests/testthat/test-include-gld-purchases.R

```diff
@@ -18,16 +20,18 @@ lmdb <- tibble::tribble(
   dplyr::bind_cols(constants)
 
 expected <- tibble::tribble(
-  ~atc,
-  "A10abc",
-  "A10",
-  "A10123",
+  ~atc, ~is_insulin_gld_code, ~is_non_insulin_gld_code,
+  "A10Abc", TRUE, FALSE,
+  "A10", FALSE, TRUE,
+  "A10B", FALSE, FALSE,
+  "A10AE5", FALSE, FALSE,
+  "A10123", FALSE, TRUE
```

Comment:
```suggestion
  ~atc, ~is_insulin_gld_code, ~is_non_insulin_gld_code,
  "A10Abc", TRUE, FALSE,
  "A10", NA, NA,
  "A10B", FALSE, TRUE,
  "A10AE5", FALSE, TRUE,
  "A10123", FALSE, FALSE
```
The atc value of the first and last lines are invalid (atc values are always these 7 digits: alpha, num, num, alpha, alpha, num, num), so they should perhaps be NA + NA, but it should never happen in the real data, so I think it's fine to keep the top one, but maybe remove the bottom one?

Reply:
The atc values don't need to be valid for these tests. But I'll update the value above to match your change.